itez/beneficiary/models.py

```diff
@@ -1,38 +1,72 @@
 from django.db import models
-from django.db.models.deletion import PROTECT 
+from django.db.models.deletion import PROTECT
+from django.db.models.fields.related import create_many_to_many_intermediary_model
 from django.utils.translation import gettext_lazy as _
+from .utils import generate_uuid_and_agent_code
 
 
+class Beneficiary(models.Model):
+    """
+    Implements the Beneficiary properties and required methods.
+    """
+
+    MARITAL_STATUS = (
+        ("single", _("Single")),
+        ("married", _("Married")),
+        ("seperated", _("Seperated")),
+        ("divorced", _("Divorced")),
+        ("widowed", _("Widowed")),
+    )
+
+    GENDER = (("male", _("Male")), ("female", _("Female")), ("other", _("Other")))
+
+    first_name = models.CharField(_("First Name"), max_length=200)
```

Comment:
check on existing models 
object data should be top to bottom and with a left shift